v2/datastream-to-spanner/terraform/samples/monitoring-alerting/README.md

```diff
@@ -0,0 +1,63 @@
+# Live Migration Monitoring Dashboard - Terraform Module
```

Comment:
Please add this template to the list of examples here - 
https://github.com/GoogleCloudPlatform/DataflowTemplates/tree/main/v2/datastream-to-spanner/terraform/samples#list-of-examples